src/chrome/content/rules/Sci-hub.xml

```diff
@@ -1,13 +1,9 @@
 <!--
 	Invalid certificate:
 		www.sci-hub.bz
-		sci-hub.io (broken chain)
-		www.sci-hub.io (broken chain)
-
-Mirrors of the documents live on their own subdomain where https is refused. Examples:
```

Comment:
This comment is still true

Reply:
The refused part is still true. But those subdomains are not used anymore from what I understand. See my full commit message for more info.